tensorflow/lite/kernels/elementwise.cc

```diff
@@ -107,6 +116,59 @@ TfLiteStatus LogicalNotEval(TfLiteContext* context, TfLiteNode* node) {
   return EvalLogical(context, node, [](bool v) { return !v; });
 }
 
+inline TfLiteStatus EvalOptimized(TfLiteContext* context, TfLiteNode* node,
+                                  void func(const RuntimeShape&, const float*,
+                                            const RuntimeShape&, float*)) {
+  const TfLiteTensor* input = GetInput(context, node, kInputTensor);
+  TfLiteTensor* output = GetOutput(context, node, kOutputTensor);
+  func(GetTensorShape(input), GetTensorData<float>(input),
+       GetTensorShape(output), GetTensorData<float>(output));
+  return kTfLiteOk;
+}
+
+template <KernelType type>
+TfLiteStatus FloorEval(TfLiteContext* context, TfLiteNode* node) {
+  if (type == kReference) {
+    return EvalNumeric(context, node, [](float f) { return std::floor(f); });
+  } else if (type == kGenericOptimized) {
+    return EvalOptimized(context, node, optimized_ops::Floor);
+  } else {
+    return kTfLiteError;
+  }
+  return kTfLiteOk;
```

Comment:
No need for return kTfLiteOk

Reply:
Alternatively, use a switch, with no default case, which will ensure you always handle the path. Same below.